tests/test_views.py

```diff
@@ -0,0 +1,78 @@
+from django.test import TestCase
+from django.urls import reverse
+from taxi.models import Manufacturer, Driver, Car
+from django.contrib.auth import get_user_model
+
+
+class ManufacturerTestView(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser", password="testpassword"
+        )
+        Manufacturer.objects.create(name="BMW",
+                                    country="Germany")
+
+    def test_manufacturer_authenticated_user(self):
+        self.client.login(username="testuser",
+                          password="testpassword")
+        response = self.client.get(reverse("taxi:manufacturer-list"))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response,
+                                "taxi/manufacturer_list.html")
+
+    def test_manufacturer_unauthenticated_user(self):
+        response = self.client.get(
+            reverse("taxi:manufacturer-list"))
+        self.assertEqual(response.status_code, 302)
+
+
+class DriverTestView(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser", password="<PASSWORD>"
+        )
+        Driver.objects.create(
+            username="alex_1993",
+            first_name="Alex",
+            last_name="Troianovskiy",
+            email="test@gmail.com",
+            license_number="ART12321",
+        )
+
+    def test_driver_authenticated_user(self):
+        self.client.login(username="testuser",
+                          password="<PASSWORD>")
```

Comment:
The placeholder '<PASSWORD>' should be replaced with the actual password used during the user creation in the setUp method. This ensures consistency and correctness when logging in the test.